package.json

```diff
@@ -51,7 +51,7 @@
   },
   "dependencies": {
     "@atproto-labs/api": "^0.12.8-clipclops.0",
-    "@atproto/api": "^0.12.5",
```

Comment:
We should not merge this update unless we also plan to get movable following feed in in the same release. Once we merge this, anyone on the TF will also start diverging from v1->v2 feeds prefs.

If we need to get this in though, there are options, just need to discuss.

Reply:
Perfect, we were wondering what that was and were going to hold off anyway so good to know. Thank you!